### PR TITLE
Added Hindi Locale

### LINKED ISSUE #13664: Add Locale hi-IN (Hindi)
- [x] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### What problem does this feature solve?
Add support for most widely used Language in India and Nepal

### What does the proposed API look like?
Add Locale strings for Hindi will add Hindi support to AntD Library

### PR DESCRIPTION
**Pull request by aashutoshrathi**
First of all, thank you for your contribution! :-)

Please makes sure that these checkboxes are checked before submitting your pull request, thank you!
Fixes #13664 

* [x] Make sure that you propose pull request to right branch: bugfix for `master`, feature for branch `feature`.
* [x] Make sure that you follow antd's [code convention](https://github.com/ant-design/ant-design/wiki/Code-convention-for-antd).
* [x] Run `npm run lint` and fix those errors before submitting in order to keep consistent code style.
* [x] Rebase before creating a pull request to keep commit history clear.
* [x] Add some descriptions and refer relative issues for you pull request.

Extra checklist:

**if** *isBugFix* **:**

  * [x] Make sure that you add at least one unit test for the bug which you had fixed.

**elif** *isNewFeature* **:**

  * [x] Update API docs for the component.
  * [x] Update/Add demo to demonstrate new feature.
  * [x] Update TypeScript definition for the component.
  * [x] Add unit tests for the feature.

### CODE REVIEW

**Review by rahgurung — APPROVED** [review]

**Comment by afc163** [comment]
`    Cannot find module 'rc-pagination/lib/locale/hi_IN' from 'hi_IN.tsx'`

File: components/time-picker/locale/hi_IN.tsx [hunk 1]
```diff
@@ -0,0 +1,5 @@
+const locale = {
+  placeholder: 'समय का चयन करें',
```
[the remark this answers is not in the document]
Comment (aashutoshrathi) [reply components/time-picker/locale/hi_IN.tsx]: I updated them

**Comment by aashutoshrathi** [comment]
@afc163 Should I send PR there?

**Comment by aashutoshrathi** [comment]
@afc163 Here we go: https://github.com/react-component/pagination/pull/158

**Comment by aashutoshrathi** [comment]
@afc163 We needed this too https://github.com/react-component/calendar/pull/466 :laughing:

**Comment by afc163** [comment]
`rc-calendar@9.9.2`

**Comment by aashutoshrathi** [comment]
@afc163 Is it normal for these 3 tests to fail?

**Comment by gurungrahul2** [comment]
> @afc163 Is it normal for these 3 tests to fail?

nah , bro ! read comments.

**Comment by aashutoshrathi** [comment]
@gurungrahul2 Can you check what I missed?

File: package.json [hunk 2]
```diff
@@ -102,7 +102,9 @@
     "ansi-styles": "^3.2.1",
     "antd-theme-generator": "^1.1.4",
     "antd-tools": "^6.4.4",
+    "babel-core": "^7.0.0-bridge.0",
     "babel-eslint": "^10.0.1",
+    "babel-jest": "^23.4.2",
```
Comment (afc163) [inline package.json]: Not need to change `dependencies`

Comment (aashutoshrathi) [reply package.json]: Getting this when removing them:
```
Cannot find module 'babel-core'
```

Comment (afc163) [reply package.json]: feature branch works well

image

**Comment by aashutoshrathi** [comment]
@afc163 It's ready now

**Pull request merged**